### PR TITLE
Update index.json

### PR DESCRIPTION
**Pull request by GarthConboy**
Add PlayBooks Audiobook Manifest processing results.

### CODE REVIEW

File: audiobooks/manifest_processing/reports/PlayBooks.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "$name"        : "PlayBooks",
+    "$description" : "Google Play Books - Results from implementing an audiobook processor in the content ingestion pipeline.",
+    "a5.01"        : true,
```
Comment (iherman) [inline audiobooks/manifest_processing/reports/PlayBooks.json:5]: Is there a web reference to the pipeline that could be used here (using the `$href` key)? It just makes the final report sexier :-)

**Review by iherman — APPROVED** [review]

File: audiobooks/manifest_processing/reports/PlayBooks.json (hunk 1 above)
Comment (GarthConboy) [reply audiobooks/manifest_processing/reports/PlayBooks.json:5]: The front door to the partner center is "https://play.google.com/books/publish/" -- would that be of interest?

Comment (iherman) [reply audiobooks/manifest_processing/reports/PlayBooks.json:5]: Sorry for the late answer; I think the answer is yes.

Comment (GarthConboy) [reply audiobooks/manifest_processing/reports/PlayBooks.json:5]: No problem!  $href added.

**Review by iherman — APPROVED** [review]

**Comment by iherman** [comment]
@wareid if you approve this, go ahead and merge it!

**Pull request merged**